src/applications/vaos/utils/constants.js

```diff
@@ -374,3 +374,8 @@ export const ELIGIBILITY_REASONS = {
   noMatchingClinics: 'noMatchingClinics',
   error: 'error',
 };
+
+export const CANCELLATION_REASONS = {
+  pat: 'pat',
```

Comment:
I assume `pat` and `prov` are used in the VAOS service to indicate `patient` and `provider`? I feel like it would make sense in this constant for the properties to better indicate what the values mean, so maybe:

```
{
  patient: 'pat',
  provider: 'prov',
}
```

Reply:
Agreed. I wanted to change it but I wasn't sure what the abbreviation were.